rep-2010.rst

```diff
@@ -231,47 +257,77 @@ These topics are vulnerable to undesired actors listening in on service communic
 Luckily, we can leverage the existing security feature for topics in ROS 2 (see SROS 2 [7]_).
 Any existing tooling for aiding users in setting up ROS security should consider the new service event topics (e.g. NoDL [8]_).
 
-Command line interface
-----------------------
-
-The existing ``ros2 service`` tool can be extended using an ``echo`` keyword to monitor service events.
-Internally, it would subscribe to the hidden topics and echo them.
-The existing command line parameters for topics can be extended to be used with this ``echo`` verb, along with new
-arguments on to filter message content and analyse delays.
 
 Rationale
 =========
 
+
+Using ROS Parameters for configuration
+--------------------------------------
+ROS parameters provide a well-known and supported way of interacting with node-level settings at various points in a node's lifecycle.
+An user may want to avoid the overhead of publishing service event messages during normal operation,
```

Comment:
minor thing but **A user** instead of **An user**

Reply:
Right. English. Vowel sound